ST_stats.h

```diff
@@ -0,0 +1,50 @@
+/* Chandler Haukap- This file defines all structs used in ST_stats.c.
+ * It was created to allow the gridded code to instanciate accumulators
+ * without including ST_stats.c. 
+ * 
+ * To read the documentation related to creating this struct see issues
+ * #259 and #266 on GitHub */
+
+#ifndef STATS_STRUCT_DEF
+#define STATS_STRUCT_DEF
+
+#include "ST_defines.h"
+
+/* Basic struct that holds average, sum of differences squared, standard deviation
+ * and number of entries. This is enough information to calculate running values without
+ * storing raw data. If you would like to create a new accumulator do not use this
+ * struct. Instead, use StatType.*/
+struct accumulators_st {
+  double ave, sum_dif_sqr, sd;
+  unsigned long nobs;
+};
+
+/* Accumulator along with the RGroup or Species name. */
+struct stat_st {
+  char *name; /* array of ptrs to names in RGroup & Species */
+  struct accumulators_st *s;
+} typedef StatType;
+
+/* Struct for wildfire and prescribed fire stats. */
+struct fire_st {
+  int *wildfire;
+  int **prescribedFire;
+} typedef FireStatsType;
+
+/*----------------------- Exported Functions ---------------------------------- */
+void stat_Collect( Int year ) ;
+void stat_Collect_GMort ( void ) ;
+void stat_Collect_SMort ( void ) ;
+void stat_Output_YrMorts( void ) ;
+void stat_Output_AllMorts( void) ;
+void stat_Output_AllBmass(void) ;
+void stat_Output_Seed_Dispersal(const char * filename, const char sep, Bool makeHeader);
+void stat_free_mem( void );
+void stat_Copy_Accumulators(StatType* newDist, StatType* newPpt, StatType* newTemp, StatType* newGrp, 
+                            StatType* newGsize, StatType* newGpr, StatType* newGmort, StatType* newGestab, 
```

Comment:
Unfortunately this isn't a typo. It's just a (poor) naming choice that was made in ST_stats.c. Grp is the group biomass and Gpr is the group PR. 

Personally I think programmers should never use acronyms and abbreviations in variable names because of confusing thinks like this. Even though changing `Grp` to `groupBiomass` makes the name longer most IDEs will recognize what you are trying to type within three characters and automatically fill it in.

But, I think the naming convention issue would be better addressed in another branch. For now I understand what they mean and the function works.

"There are only two hard things in Computer Science: cache invalidation and naming things." - Phil Karlton

Reply:
@chaukap thanks for the clarification! I agree. And I appreciate the quote as well. :)